icons/svg/item-properties/magical.svg

```diff
@@ -0,0 +1 @@
+<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg" viewBox="20 20 492 492"><g  fill="#000" transform="translate(0,0)"><path d="M403.563 16.875c-34.11 33.744-53.907 25.412-89.094 4.844 31.052 27.848 44.703 40.722 9.25 76.124 32.57-26.112 44.422-29.22 85.342-15-23.523-25.805-21.323-36.88-5.5-65.97zm90.312 24.406C463.23 68.004 425.365 93.7 386.687 113c1.152 61.455-37.802 117.82-99.125 165.406C221 330.06 127.345 372.848 24.344 404.28v41.22c15.817-3.243 32.09-6.896 48.656-11.03-2.655 14.497-11.253 27.387-28.938 39.81 43.158-14.862 52.446-1.325 76.188 18.814-7.644-26.835-5.256-36.344 34.625-58.438-32.328 9.304-48.716 5.836-63.03-5.094 74.51-20.002 153.373-49.706 221.25-93.625 6.362 31.056 5.325 56.495-36.19 94.282 67.325-35.667 96.207-34.27 130.97 7.155-34.905-53.113-30.953-75.32 14.063-123.97-42.665 33.767-70.8 30.987-94.063 12.626.615-.427 1.23-.85 1.844-1.28 90.41-63.473 157.526-153.397 164.155-283.47zM240.75 70.97c4.865 36.552-4.39 56.492-23.938 78.53 31.65-24.863 39.97-22.61 71.157-23-28.584-13.45-36.21-20.397-47.22-55.53zm-110.78 40.686c-27.298 60.18-58.556 70.662-107.626 70.125 59.78 20.1 64.886 52.96 69.906 106.25 19.34-58.957 36.19-76.01 78.72-58.217-41.78-31.838-45.743-48.97-41-118.157zm237.936 10.188c-30.864 13.607-61.587 22.598-88.937 24.28-2.278 16.52-11.623 32.867-25.19 49-15.767 18.755-37.557 37.463-62.78 55.69-49.014 35.414-110.962 68.736-166.656 94.28v39.625c99.33-30.83 189.144-72.456 251.78-121.064 55.735-43.25 89.362-90.922 91.782-141.812zm99.53 98.97c-7.594 35.52-13.903 44.526-46.655 65.31 35.874-8.104 51.316-1.513 74.533 15.032-22.447-30.02-28.5-41.96-27.875-80.344zM259.75 400.78c-30 43.11-55.372 48.865-98.656 52.22 51.72 7.82 65.302 14.218 95.875 41.094-12.9-35.393-15.344-48.14 2.78-93.313z"></path></g></svg>
```

Comment:
Setting the fill to be `var(--icon-fill, #000)` will allow easily recoloring of the icon.

```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg" viewBox="20 20 492 492"><g style="fill: var(--icon-fill, #000);" transform="translate(0,0)"><path d="M403.563 16.875c-34.11 33.744-53.907 25.412-89.094 4.844 31.052 27.848 44.703 40.722 9.25 76.124 32.57-26.112 44.422-29.22 85.342-15-23.523-25.805-21.323-36.88-5.5-65.97zm90.312 24.406C463.23 68.004 425.365 93.7 386.687 113c1.152 61.455-37.802 117.82-99.125 165.406C221 330.06 127.345 372.848 24.344 404.28v41.22c15.817-3.243 32.09-6.896 48.656-11.03-2.655 14.497-11.253 27.387-28.938 39.81 43.158-14.862 52.446-1.325 76.188 18.814-7.644-26.835-5.256-36.344 34.625-58.438-32.328 9.304-48.716 5.836-63.03-5.094 74.51-20.002 153.373-49.706 221.25-93.625 6.362 31.056 5.325 56.495-36.19 94.282 67.325-35.667 96.207-34.27 130.97 7.155-34.905-53.113-30.953-75.32 14.063-123.97-42.665 33.767-70.8 30.987-94.063 12.626.615-.427 1.23-.85 1.844-1.28 90.41-63.473 157.526-153.397 164.155-283.47zM240.75 70.97c4.865 36.552-4.39 56.492-23.938 78.53 31.65-24.863 39.97-22.61 71.157-23-28.584-13.45-36.21-20.397-47.22-55.53zm-110.78 40.686c-27.298 60.18-58.556 70.662-107.626 70.125 59.78 20.1 64.886 52.96 69.906 106.25 19.34-58.957 36.19-76.01 78.72-58.217-41.78-31.838-45.743-48.97-41-118.157zm237.936 10.188c-30.864 13.607-61.587 22.598-88.937 24.28-2.278 16.52-11.623 32.867-25.19 49-15.767 18.755-37.557 37.463-62.78 55.69-49.014 35.414-110.962 68.736-166.656 94.28v39.625c99.33-30.83 189.144-72.456 251.78-121.064 55.735-43.25 89.362-90.922 91.782-141.812zm99.53 98.97c-7.594 35.52-13.903 44.526-46.655 65.31 35.874-8.104 51.316-1.513 74.533 15.032-22.447-30.02-28.5-41.96-27.875-80.344zM259.75 400.78c-30 43.11-55.372 48.865-98.656 52.22 51.72 7.82 65.302 14.218 95.875 41.094-12.9-35.393-15.344-48.14 2.78-93.313z"></path></g></svg>
```